test/unit/specs/components/wallet/PageTransactions.spec.js

```diff
@@ -48,15 +250,14 @@ describe(`PageTransactions`, () => {
     $store = {
       commit: jest.fn(),
       dispatch: jest.fn(),
-      getters
+      getters: JSON.parse(JSON.stringify(getters)) // clone so we don't overwrite by accident
```

Comment:
are the `.parse` and `.stringify` necessary?

Reply:
this is the safes and quickest way I know to clone an object